src/ConfigurationTrait.php

```diff
@@ -166,9 +166,15 @@ public function setInput(InputInterface $input)
     public function bootstrap(InputInterface $input, OutputInterface $output)
     {
         parent::bootstrap($input, $output);
-        $connection = ConnectionManager::get($this->getConnectionName($input));
-        $pdo = $this->getManager()->getEnvironment('default')->getAdapter()->getConnection();
-        $connection->driver()->connection($pdo);
+        $name = $this->getConnectionName($input);
+        ConnectionManager::alias($name, 'default');
+        $connection = ConnectionManager::get($name);
```

Comment:
If you invoke the migrations shell passing a custom connection argument, you would expect that using any table class would automatically default to that custom connection instead of another.